code/modules/mob/living/announcer.dm

```diff
@@ -0,0 +1,121 @@
+// This mob is a lie. Similar to /mob/abstract it's not really supposed to do much.
+// It is used as a fake announcer so that we have a lightweight mob to pass to radio broadcasts.
+// To use it simply instantiate it, store a reference to it - it can be reused, so no reason
+// to delete it immediately. Just before a broadcast call PrepareBroadcast with desired arguments,
+// then right after the broadcast call ResetAfterBroadcast so you can re-use the mob without issues next time.
+
+/mob/living/announcer
+	name = "Announcer"
+	voice_name = "synthesized voice"
+	accent = ACCENT_TTS
+
+	icon = 'icons/mob/AI.dmi'
+	icon_state = "ai"
+	gender = NEUTER
+
+	status_flags = GODMODE|NO_ANTAG|NOFALL
+	invisibility = INVISIBILITY_ABSTRACT
+	mob_thinks = FALSE
+
+	universal_understand = TRUE
+	can_have_vision_cone = FALSE
+	tesla_ignore = TRUE
+	stop_sight_update = TRUE
+	density = FALSE
+	burn_mod = 0
+	brute_mod = 0
+
+/mob/living/announcer/Initialize()
+	// we specifically do not call parent Initialize here because this mob should not need it and its point is to be lightweight
+	SHOULD_CALL_PARENT(FALSE)
+	if(initialized)
+		crash_with("Warning: [src]([type]) initialized multiple times!")
+	initialized = TRUE
+
+	for(var/K in all_languages)
+		add_language(K)
+	default_language = all_languages[LANGUAGE_TCB]
+
+/mob/living/announcer/Destroy()
+	// again not calling parent because it shouldn't be needed and makes this faster
+	SHOULD_CALL_PARENT(FALSE)
+	for(var/C in contents)
```

Comment:
Destroy has important logic to remove weakrefs, events, timers and so on. I don't think this is worth the millisecond speedup.

Reply:
If you intend to keep this, which I don't reccomend - add a destroy hint return.